src/blocks/gallery-stacked/index.js

```diff
@@ -35,7 +35,7 @@ const settings = {
 	title: _x( 'Stacked', 'block name' ),
 	description: __( 'Display multiple images in an single column stacked gallery.' ),
 	icon,
-	keywords: [	_x( 'gallery', 'block keyword' ), _x( 'photos', 'block keyword' ) ],
+	keywords: [	_x( 'gallery', 'block keyword' ), _x( 'photos', 'block keyword' ), _x( 'lightbox', 'block keyword' ) ],
```

Comment:
This change is not related to the PR and instead looks like it should be moved into PR #840.